cpu/stm32/include/cpu_conf_stm32_common.h

```diff
@@ -66,6 +62,31 @@ static const i2c_timing_param_t timing_params[] = {
         .sdadel = 0x0,      /* t_SDADEL = 0ns   */
         .scldel = 0x1,      /* t_SCLDEL = 250ns */
     }
+#elif defined(CPU_FAM_STM32L4) || defined(CPU_FAM_STM32F7) || \
```

Comment:
Open question: should I also include f0 and f3 in this list ? And then drop f0 and f3 specific parameters. I haven't tried I2C on these platforms with this PR and with these parameters but I think that's worth.

Reply:
Please do

Reply:
Done, with some extra cleanups: the `cpu_conf_common.h` was not needed anymore. I moved its content in `cpu_i2c.h`, guarded by the same CPU family check and removed the file.
Let's see if that passes Murdock.